Sources/StreamChatUI/CommonViews/CheckboxControl/ChatCheckboxControl.swift

```diff
@@ -5,7 +5,8 @@
 import StreamChat
 import UIKit
 
-open class ChatMessageComposerCheckmarkControl: _Control, AppearanceProvider {
+/// A view to check/uncheck an option.
+open class ChatCheckboxControl: _Control, AppearanceProvider {
```

Comment:
I don't event think we need the `Chat` prefix here

Reply:
We need to write down the pattern when to use the `Chat` prefix. As you can see below, right now, all the components have the `Chat` prefix, only `TypingView` doesn't have it. For me, or we either write the prefix in all views, or in the UI SDK, we don't use the prefix at all, because if we use it in some and other's we don't, will be confusing.
![image](https://user-images.githubusercontent.com/12814114/116677459-24ab4e00-a9a0-11eb-8040-8b1fb92c8a72.png)